src/middlewared/middlewared/plugins/update_/utils.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding=utf-8 -*-
+import itertools
+import re
+
+SCALE_MANIFEST_FILE = "/data/manifest.json"
+SCALE_UPDATE_SERVER = "http://pivnoy.thelogin.ru/scale"
```

Comment:
@kmoore134 what should this be? http://download.truenas.com/scale

Reply:
For updates? Lets use: https://update.freenas.org/scale/ for now. 

Eventually we will do a DNS re-direct to update.truenas.org though, and we can update our location.